### PR TITLE
feat: add color printing utilities

### PR DESCRIPTION
**Pull request by almas1992**
## 📑 Description

Package color provides utilities for rendering and printing colorized text using color tags.

The color tags are automatically parsed and rendered.

Supported Features:
1. Style Tags:
   Use predefined style tags to format text. Example:
   `color.Print("<suc>he</><comment>llo</>, <cyan>wel</><red>come</>")`

2. Custom Color Attributes:
   Use specific color attributes for more granular control. Example:
  ` color.Println("<fg=11aa23>he</><bg=120,35,156>llo</>, <fg=167;bg=232>wel</><fg=red>come</>")`

image

Color tag more detail in https://gookit.github.io/color/#/?id=html-like-tag-usage

Closes https://github.com/goravel/goravel/issues/?

@coderabbitai summary

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

**Comment by almas1992** [comment]
Review Ready

File: support/color/print.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+/*
+Package color provides utilities for rendering and printing colorized text using color tags.
+
+The color tags are automatically parsed and rendered.
+
+Supported Features:
+1. Style Tags:
+   Use predefined style tags to format text. Example:
+   color.Print("<suc>he</><comment>llo</>, <cyan>wel</><red>come</>")
```
Comment (hwbrzzl) [inline support/color/print.go:13]: Is there a rule that describes the tags, please?

Comment (kkumar-gcc) [reply support/color/print.go:13]: Don't we have `color.Red().Println` and other methods?

Comment (almas1992) [reply support/color/print.go:13]: > Is there a rule that describes the tags, please?

https://gookit.github.io/color/#/?id=html-like-tag-usage

Comment (almas1992) [reply support/color/print.go:13]: > Don't we have `color.Red().Println` and other methods?

It’s not just about simple colors like red or green. We also need to style text with bold, italic, underline, and use a wider range of RGB colors. For this, using gookit/color with its flexible color tag configuration is highly recommended.

**Review by hwbrzzl — APPROVED** [review]
Great 👍

**Pull request merged**